examples/helia-browser-react-verified-fetch/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "helia-browser-react-verified-fetch",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@helia/verified-fetch": "0.0.0-3851fe2",
```

To get the latest RC until every time until we publish v1

```suggestion
    "@helia/verified-fetch": "next",
```